NEWS.md

```diff
@@ -7,6 +7,12 @@ extension developers if they have relied on internals that have been changed.
 This release also sees the addition of Hiroaki Yutani (@yutannihilation) to the 
 core developer team.
 
+Two patches (#2996 and #3050) fixed minor rendering problems. In most cases, the
```

Comment:
Maybe put under `## Breaking changes`?

Reply:
That works for me. I remember we had some discussion in the past about what exactly a breaking change is, but it feels like a breaking change to me. I also added a bullet point where I'm trying to express that ggplot2 is now somewhat stricter and now creates errors or warnings where previously it would just create some arbitrary output. Not sure I'm totally happy with this. I'm referring to issues such as #3316, #3305, #3318.

Reply:
I added the specific examples to the bullet point. I think that makes it better.

Reply:
I'd say that breaking a revdep check is sufficient (but not necessary) for a change to be breaking.